static/inspect.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Raw Vector Tile Data View</title>
+  <link rel="stylesheet" type="text/css" href="/maplibre-gl.css" />
+  <link rel="stylesheet" type="text/css" href="/maplibre-gl-inspect.css" />
+  <script>if (typeof Symbol !== 'undefined') { document.write('<script src="https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js"><\/script>'); } else { document.write('<script src="/maplibre-gl-compat.js"><\/script>'); }</script>
+  <script>if (typeof Symbol !== 'undefined') { document.write('<script src="https://github.com/acalcutt/maplibre-gl-inspect/releases/download/v1.4.6/maplibre-gl-inspect.min.js"><\/script>'); } else { document.write('<script src="/maplibre-gl-inspect-compat.min.js"><\/script>'); }</script>
```

Comment:
Since you’re pulling in the latest version of GL JS, you might want to keep this library up to date as well. The current latest version is [v1.6.3](https://github.com/maplibre/maplibre-gl-inspect/releases/tag/v1.6.3).